Automattic/jetpack
# Search: Remove use of wrong props withHeader and withFooter for AdminPage component in Dashboard

We're trying to pass `withHeader={true}` and `withFooter={true}` to `<AdminPage/>` but the actual props are called `showFooter` and `showHeader`. Also, the default for these props is true so we don't need to pass them for the purpose here.

Fixes nothing.

#### Changes proposed in this Pull Request:

* Updates Upsell component to not pass options for footer and header

#### Other information:


#### Jetpack product discussion
None

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Build search package
* Visit the dashboard and confirm you see header and footer
